### ISSUE TITLE
ATL-01: Remove model.train()/eval() distinction [CS-10]

### ISSUE THREAD

**Issue by toddwbucy** [toddwbucy/Atlas-MAG_OmegaRule#60]
## Code Smell Reference
**CS-10**: No `model.train()` / `model.eval()` distinction in the NL ontology.

## Problem
In `src/model/blocks.py`, the TTL (Test-Time Learning) inner loop update is gated behind `self.training`:

```python
if self.ttl_enabled and self.training:
```

This creates a train/eval mode distinction — TTL runs during "training" but not during "inference". The NL ontology rejects this: the system should behave identically regardless of mode. Memory is self-modifying at test time; that's the entire point.

## Fix
Remove the `self.training` guard:

```python
if self.ttl_enabled:
```

## Files Affected
- `src/model/blocks.py` (line ~245: remove `and self.training`)
- `tests/test_ttl.py` (update tests to verify TTL runs in both modes)

## Acceptance Criteria
- [ ] TTL runs regardless of `model.train()` or `model.eval()` state
- [ ] Test verifies TTL produces stats in both modes
- [ ] All existing tests pass

**Comment by toddwbucy** [comment]
Closing: this repo is now archived as a pre-graph baseline. Ongoing work continues in NestedLearning/Atlas/ using the NL graph methodology.
